Xwt.XamMac/Xwt.Mac/PanedBackend.cs

```diff
@@ -118,25 +120,83 @@ public override void ReplaceChild (NSView oldChild, NSView newChild)
 				Widget.AddSubview (view2);
 		}
 
-		public double Position {
-			get {
-				return 0;
+		private double position;
+		public double Position
+		{
+			get
```

Comment:
Can you put the { at the end of the line, so it's "get {". That's the convention here.

Reply:
Because there's no editorconfig, these issues will just resurface. Imo, codestyle here is not important.